### PR TITLE
Script fixes for IE

### LINKED ISSUE #5043: JavaScript error on element.isClickable() and element.waitForClickable() in IE11
Hey everyone,
I am having an issue with IE11 and webdriverIO used with Browserstack. isClickable() and waitForClickable() is casing a JavaScript error and so far I was unable to find any related topics on this.
I use "webdriverio": "^5.18.6" in sync with BrowserStack browsers. Unfortunately the error log doesn't actually describe what the problem is, but if I take those methods out, it works.

`[0-0] 2020-02-19T14:14:34.693Z ERROR webdriver: Request failed due to Error: JavaScript error
    at getErrorFromResponseBody (node_modules/webdriver/build/utils.js:124:10)
    at Request._callback (node_modules/webdriver/build/request.js:111:64)
    at Request.self.callback (node_modules/request/request.js:185:22)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at Request.<anonymous> (node_modules/request/request.js:1161:10)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at IncomingMessage.<anonymous> (node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:286:20)

[0-0] 2020-02-19T14:14:36.189Z ERROR webdriver: Request failed due to Error: JavaScript error
    at getErrorFromResponseBody (node_modules/webdriver/build/utils.js:124:10)
    at Request._callback (node_modules/webdriver/build/request.js:111:64)
    at Request.self.callback (node_modules/request/request.js:185:22)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at Request.<anonymous> (node_modules/request/request.js:1161:10)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at IncomingMessage.<anonymous> (node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:286:20)

[0-0] 2020-02-19T14:14:37.282Z ERROR webdriver: Request failed due to Error: JavaScript error
    at getErrorFromResponseBody (node_modules/webdriver/build/utils.js:124:10)
    at Request._callback (node_modules/webdriver/build/request.js:111:64)
    at Request.self.callback (node_modules/request/request.js:185:22)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at Request.<anonymous> (node_modules/request/request.js:1161:10)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at IncomingMessage.<anonymous> (node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:286:20)`

**Browserstack Logfile**
@christian-bromann I provided logs from Browserstack now
[logs.txt](https://github.com/webdriverio/webdriverio/files/4225576/logs.txt)

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

closes #5043

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

n/a

### CODE REVIEW

File: packages/webdriverio/src/scripts/isElementClickable.js [hunk 1]
```diff
@@ -65,23 +69,30 @@ export default function isElementClickable (elem) {
...
+        elemsWithShadowRoot = elemsWithShadowRoot.filter(function (x) {
+            return x && x.shadowRoot && x.shadowRoot.elementFromPoint
+        })
 
         // getOverlappingElements of every element with shadowRoot
         let shadowElementsFromPoint = []
-        for (let shadowElement of elemsWithShadowRoot) {
-            shadowElementsFromPoint.push(...getOverlappingElements(elem, shadowElement.shadowRoot))
+        for (let i = 0; i < elemsWithShadowRoot.length; ++i) {
+            let shadowElement = elemsWithShadowRoot[i]
```
Comment (erwinheitzman) [inline packages/webdriverio/src/scripts/isElementClickable.js:88]: if the idea is to make it all ES5 and thus IE11 compatible then the keywords `let` and `const` cannot be used either.

Comment (christian-bromann) [reply packages/webdriverio/src/scripts/isElementClickable.js:86]: @erwinheitzman `let` and `const` are supported by IE11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

**Comment by j320** [comment]
Great news! Is there a release date for this fix?

**Comment by christian-bromann** [comment]
@j320 this was already released. Please see changelog.

**Pull request merged**